docarray/typing/tensor/tensor.py

```diff
@@ -1,6 +1,8 @@
-from typing import Union
+from typing import TYPE_CHECKING, Union
 
 from docarray.typing.tensor.ndarray import NdArray
-from docarray.typing.tensor.torch_tensor import TorchTensor
 
-Tensor = Union[NdArray, TorchTensor]
+if TYPE_CHECKING:
+    from docarray.typing.tensor.torch_tensor import TorchTensor
+
+Tensor = Union[NdArray, 'TorchTensor']
```

Comment:
Does this (`'TorchTensor'` in the Union) really work? Can we test it?

Reply:
What kind of test do you have in mind ?